src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,60 @@
+package duke.task;
+
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class for tasks created.
+ */
+public class Task {
+    protected DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MMM dd yyyy");
+    private boolean status;
```

Comment:
Perhaps you could consider changing this boolean name to something more intuitive (e.g. isDone)?

Reply:
Thanks for this suggestion!